distance_based_on_real_case.py

```diff
@@ -0,0 +1,1133 @@
+
+Drogon case:
+1. Assume all observations except RFT and PLT data can update parameters for all zones.
+   The lateral influence of an observation will limit how far laterally an observation
+   will have effect on field parameters, but vertically 
+   the field parameters with (x,y) distance within the localization range is updated.
+   For zone dependent data like RFT and PLT, they are only included in observation vector
+   for the relevant zones. It is possible to experiment with zone dependence such that
+   some observations are only able to update field parameters in one zone and not all 3 zones.
+2. The ERTBOX coordinate system is the same for all zones (same lateral definition of ERTBOX). This means
+   that coordinate transformation of position of observations is only necessary to be calculated once and
+   calculation of rho for one layer can be done once as long as all zones use the same observations.
+3. The size of the field parameters (nx * ny * nz_zone_dependent) is small enough that one usually don't have
+   to split the update of a parameter group into smaller batches in this case.
+4. The naming convention used by APS allow us to identify the zone name for each field parameter group
+   but this is in general not the case and therefore a hard-coded dict in this script define the zone per field
+   enven though this is not necessary when only using field parameters coming from APS where zone name
+   is part of the field parameter name in ERT.
+5. Number of active parameters vary from realization to realization in Therys, but not in the other two zones.
+   There are no large staircase faults that creates active/inactive field parameters from realization to realization.
+
+What Drogon case cannot test:
+1. A more efficient specification of seismic 4D data defined by a regular 2D grid aligned with the ERTBOX
+2. No observation related to regions or the reservoir as a whole.
+3. No observation of fluid contacts (they are used as parameters with a prior distribution)
+4. Drogon size of the field parameters are not very large compared with real reservoir cases.
+5. Drogon does not limit updates of 2D SURFACE parameters to observations directly related to volume, but
+   is updated by using all observations.
+6. Cannot test the effect of having active/inactive values varying from realization to realization along staircase faults.
+"""
+
+STORAGE_PATH = (
+    "/project/fmu/users/olia/drogon_20250623_11-50/"
+    "resmod/ff/25.0.0/ert/output/drogon_ahm/storage"
+)
+
+SEED = 42
+ALPHA = 1
+NLAYER_PER_BATCH = (
+    40  # 66 layers in ERTBOX for Drogon which is split into 2 batches here
+)
+# Turn on/off calculation of rho based on localization or only use rho = 1
+USE_LOCALIZATION = False
+
+# Turn on/off export of field parameters into 3D parameter files in ROFF format to be read by RMS.
+WRITE_STATS_PARAMS = True
+
+# Common definition of ERTBOX for all zones in Drogon
```

Comment:
Does the grid file specified using the `GRID` keyword in the Drogon project contain all this information?